pkgs/development/python-modules/vosk/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, vosk, requests, srt, tqdm, websockets }:
+
+buildPythonPackage rec {
+  pname = "vosk";
+  inherit (vosk) version src meta;
+
+  sourceRoot = "source/python";
+
+  postPatch = ''
+    substituteInPlace vosk/__init__.py \
+      --replace "dlldir = os.path.abspath(os.path.dirname(__file__))" 'dlldir = "${vosk}/lib"'
+  '';
+
+  propagatedBuildInputs = [
+    requests
+    srt
+    tqdm
+    websockets
+  ];
```

Comment:
How does it work [without cffi](https://github.com/alphacep/vosk-api/blob/32aa980069495b88aa79d192d1ef42ad88d0ec73/python/setup.py#L74)? I had to include it when I packaged it for https://github.com/NixOS/nixpkgs/pull/185148.